lib/modules/manager/helm-values/extract.ts

```diff
@@ -34,6 +45,7 @@ function getHelmDep({
 function findDependencies(
   parsedContent: Record<string, unknown> | HelmDockerImageDependency,
   packageDependencies: Array<PackageDependency>,
+  config?: ExtractConfig,
```

Comment:
```suggestion
  config: ExtractConfig,
```

is always passed, so fix tests